console.rst

```diff
@@ -585,7 +585,7 @@ call ``setAutoExit(false)`` on it to get the command result in ``CommandTester``
 When testing your commands, it could be useful to understand how your command
 reacts on different settings like the width and the height of the terminal.
 You have access to such information thanks to the
-:class:`Symfony\\Component\\Console\\Terminal` class:
+:class:`Symfony\\Component\\Console\\Terminal` class::
```

Comment:
Can we spot this automatically @alamirault 

Any idea?

Reply:
I think we can't or it will be very hard.

Malformed [Literal blocks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) can be confused with a lot other RST blocks.

Following regex can be a start `.+:\n\n    `, but we must exclude a lot of false positive